
sensor_msgs/PointCloud2             cloud
inliers_indices[]                   segmented_inliers       #Array of holding the indices of each array